cmake/IgnPython.cmake

```diff
@@ -12,14 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Copied from ament/ament_cmake: ament_cmake/ament_cmake_core/cmake/core/python.cmake
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.19")
+  set(PYTHON_VERSION "" CACHE STRING
```

Comment:
Could we please rename `PYTHON_VERSION` to avoid possible conflicts using that generic name? Maybe using `IGN_PYTHON_VERSION`?

Reply:
Done :)